tests/test_main.c

```diff
@@ -28,9 +26,10 @@ int main(void) {
     int number_failed;
     SRunner *sr;
 
-    sr = srunner_create(kfs_mkdir_suite());
-   // srunner_add_suite(sr, kfs_common_suite());
-   // srunner_add_suite(sr, kfs_access_suite());
+    sr = srunner_create(kfs_open_suite());
```

Comment:
don't push those comments, it used only for debug and development, we don't need this in master